modules/installation-osp-provider-network-preparation.adoc

```diff
@@ -44,7 +44,7 @@ Provider networks must be owned by the {rh-openstack} project that is used to cr
 
 * Verify that the provider network can reach the {rh-openstack} metadata service IP address, which is `169.254.169.254` by default.
 +
-Depending on your {rh-openstack} SDN and networking service configuration, you might need to provide the route when you create the subnet. For example:
+Depending on your {rh-openstack} OVN-Kubernetes network plugin and networking service configuration, you might need to provide the route when you create the subnet. For example:
```

Comment:
I guess here no need to updated.

Reply:
Yeah, I wasn't sure if this was a generic use of software defined network or the SDN plugin. Thank you.